### PR TITLE
Benchmarking in evmtool

### PR DESCRIPTION
**Pull request by shemnon**
## PR description

Add a new benchmarking sub-command to `evmtool`

### CODE REVIEW

File: besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public class NativeLibraryOptions {
           "Per default a native library is used for modexp. "
               + "If the Java implementation should be used instead, this option must be set to false",
       arity = "1")
-  private final Boolean nativeModExp = Boolean.TRUE;
+  private final Boolean nativeModExp = Boolean.FALSE;
```
Comment (daniellehrner) [inline besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java:56]: Why was this changed to `FALSE`? Shouldn't we prefer the native one per default?

File: besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public class NativeLibraryOptions {
           "Per default a native library is used for modexp. "
```
Comment (daniellehrner) [inline besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java:53]: If the change in line 56 is correct this description needs to be changed to default `false`

**Review by daniellehrner — CHANGES_REQUESTED** [review]
Clarification needed regarding native mod exp command line.

**Review by daniellehrner — APPROVED** [review]

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/benchmarks/BenchmarkExecutor.java [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    this.iterations = iterations;
+  }
+
+  /** Run benchmarks with warmup and iterations set to MATH style benchmarks. */
+  protected BenchmarkExecutor() {
+    this(MATH_WARMUP, MATH_ITERATIONS);
+  }
+
+  /**
+   * Run the benchmark with the speicific args. Execution will be done warmup + iterations times
```
Comment (macfarla) [inline ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/benchmarks/BenchmarkExecutor.java:92]:
```suggestion
   * Run the benchmark with the specific args. Execution will be done warmup + iterations times
```

**Pull request merged**